### PR TITLE
adds blog for cns on aws

### PR DESCRIPTION
**Pull request by Sterling463**
In a comment on the pull request, include the following information and check off the list.

 - [ ] Mini bio
  - Soft capped to 75 words
 - [ ] Desired date of publishing
  - Make sure this matches the `date` in the front matter
 - [ ] Social media contact info
 - [ ] Redaction: (yes|no)
  - Allow redaction, editing or structure changes by technical writer staff
 - [ ] Comments: (yes|no)
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by Sterling463** [comment]
build me

**Comment by rackernexus** [comment]
Build 1002 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-4a73bc5fb/). valid for 14 days

File: _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    - General
+metaTitle: "Cloud Native Security for AWS comes to Rackspace "
+metaDescription: "The Rackspace Service Blocks for Cloud Native Security (CNS) integrates directly with the AWS Security Hub,
+pulling threat information and alerts into your AWS environment."
+ogTitle: "Cloud Native Security for AWS comes to Rackspace"
+ogDescription: "The Rackspace Service Blocks for Cloud Native Security (CNS) integrates directly with the AWS Security Hub,
+pulling threat information and alerts into your AWS environment."
+---
+
+Amazon Web Services (AWS) has more Cloud Native Cybersecurity tools available than ever,
```
Comment (maximrozin) [inline _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md:19]: I would replace "Cloud Native Cybersecurity tools" with "native  security products"

File: _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+but many businesses lack the expertise to quickly analyze and respond to this data. Rackspace
+is the first consulting managed security service provider (MSSP) to partner with the AWS Security Hub.
```
Comment (maximrozin) [inline _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md:21]: Remove "the" before "AWS Security Hub"

File: _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+The Rackspace Service Blocks for Cloud Native Security (CNS) integrates directly with the AWS Security Hub,
```
Comment (maximrozin) [inline _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md:22]: The Rackspace Service Blocks for Cloud Native Security (CNS) integrates directly with the AWS Security Hub, pulling threat information and alerts into your AWS environment. --> The Rackspace Service Block for Cloud Native Security (CNS) integrates directly with AWS Security Hub, ingesting threat information and alerts from your AWS environment.

File: _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+pulling threat information and alerts into your AWS environment.
+
+<!-- more -->
+Certified security experts in the global Rackspace Security Operations Center (SOC)
+offers you around-the-clock managed security services for 3 AWS security products:
```
Comment (maximrozin) [inline _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md:27]: offers --> offer

File: _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+### Amazon GuardDuty
+
+Amazon GuardDuty is a threat detection service that uses machine learning, anomaly detection,
+and integrated threat intelligence to collect and aggregate account and network activities to identify
+potential threats. Rackspace integrates and configures Amazon GuardDuty for your accounts and continuously
+analyzes these findings. Certified security experts help you understand and respond to this information.
+
+### IAM Access Analyzer
+
+The IAM Access Analyzer identifies resource-based policies that would allow external access to your
```
Comment (maximrozin) [inline _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md:42]: Remove "The" before "IAM Access Analyzer"

File: _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+environment. Rackspace monitors this information, validating any changes, and addressing issues as necessary.
+
+### AWS Security Hub
+
+The AWS Security Hub enables aggregation of findings from member accounts and from other AWS Security services,
+like GuardDuty and IAM Access Analyzer. Rackspace provides you full visibility to this information through an
+easy to understand interface. Rackspace CNS enables and configures the AWS Security Hub for all your accounts
+and regions in scope.
```
Comment (maximrozin) [inline _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md:50]: The AWS Security Hub enables aggregation of findings from member accounts and from other AWS Security services, like GuardDuty and IAM Access Analyzer. Rackspace provides you full visibility to this information through an easy to understand interface. Rackspace CNS enables and configures the AWS Security Hub for all your accounts and regions in scope. --> AWS Security Hub enables aggregation of findings from member accounts and from other AWS security products, like GuardDuty and IAM Access Analyzer. It also provides full visibility to this information through an easy to understand interface. Rackspace CNS enables and configures AWS Security Hub for all your accounts and regions in scope.

File: _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+- **24x7x365 Monitoring & Response.** Get around-the-clock monitoring, advanced analysis and threat remediation from certified security experts in the Rackspace SOC.
+
+- **Comprehensive Security Visibility.** Data from your supported AWS security tools will be integrated into our Security Information and Event Management (SIEM) system, for a comprehensive view of threats across all of your environments with Rackspace Managed Security Services.
+
+- **AWS Security Expertise.** Rackspace is the first pure Managed Security Service Provider (MSSP) Partner for AWS Security Hub, offering consulting services for AWS native security tools that you can’t get anywhere else.
+
+
+## Get started today
+
+Visit [www.rackspace.com](https::/www.rackspace.com) and click Sales Chat to start learning more about how Rackspace Cloud Native Security for AWS can help your business meet its goals.
```
Comment (maximrozin) [inline _posts/2020-04-16-cloud-native-security-for-aws-comes-to-rackspace.md:65]: Change the link to: https://www.rackspace.com/managed-aws/capabilities/security

click Sales Chat --> request a quote or chat with us

**Pull request merged**